### ISSUE TITLE
log4j lib still exists in 0.5.3 causing Proguard failing

### ISSUE THREAD

**Issue by jaredzhang** [uber/rides-android-sdk#54]
```
+--- com.uber.sdk:rides-android:0.5.3
|    +--- com.android.support:appcompat-v7:23.0.1 -> 24.2.0 (*)
|    \--- com.uber.sdk:core-android:0.5.3
|         +--- com.android.support:appcompat-v7:23.0.1 -> 24.2.0 (*)
|         +--- com.google.code.findbugs:jsr305:1.3.9
|         \--- com.uber.sdk:rides:0.5.2
|              +--- com.squareup.okhttp3:logging-interceptor:3.2.0
|              |    \--- com.squareup.okhttp3:okhttp:3.2.0 (*)
|              +--- org.slf4j:slf4j-log4j12:1.7.5
|              |    +--- org.slf4j:slf4j-api:1.7.5
|              |    \--- log4j:log4j:1.2.17

```
#31

**Comment by tyvsmith** [comment]
Thanks for reporting. It looks like the Gradle dependency exclusion isn't being implemented in the Pom generator. For a workaround, look at the exclusion of log4j that's being done in [here](https://github.com/uber/rides-android-sdk/blob/master/core-android/build.gradle#L160)

**Comment by jaredzhang** [comment]
it works, thx

**Comment by itstexter** [comment]
Fixed in [v0.5.4](https://github.com/uber/rides-android-sdk/releases/tag/v0.5.4)
